### PR TITLE
MCD migration - Update SFBO phase description

### PR DESCRIPTION
**Pull request by heikobischoff**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/ms-cloud-germany-transition-phases.md [hunk 1]
```diff
@@ -224,16 +223,24 @@ In this phase, Skype for Business will be migrated to Microsoft Teams. Existing
...
 
 ```powershell
 Import-Module MicrosoftTeams
 $userCredential = Get-Credential
-Connect-MicrosoftTeams -Credential $userCredential -OverridePowershellUri "https://admin4E.online.lync.com/OcsPowershellOAuth"
+Connect-MicrosoftTeams -Credential $userCredential
 ```
 
+### Known limitations until finalizing Azure AD migration
+Microsoft Teams is leveraging features of Azure AD. While the migration of Azure AD is not completed, some features of Microsoft Teams are not completely available. After phase 9, when the migration of Azure AD will be finalized, the following features become fully available:
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition-phases.md]:
```suggestion
Microsoft Teams is leveraging features of Azure AD. While the migration of Azure AD is not completed, some features of Microsoft Teams are not fully available. After phase 9, when the migration of Azure AD has been finalized, the following features become fully available:
```

File: microsoft-365/enterprise/ms-cloud-germany-transition-phases.md [hunk 2]
```diff
@@ -224,16 +223,24 @@ In this phase, Skype for Business will be migrated to Microsoft Teams. Existing
...
+
+- Apps cannot be managed in the Microsoft Teams admin center.
+- New Teams can be created in the Microsoft Teams client only unless the Teams administrator constrained permissions to create new Teams for users. New Teams cannot be created in the Microsoft Teams admin center.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition-phases.md]:
```suggestion
- New teams can be created in the Microsoft Teams client only unless the Teams administrator has limited the permissions for users to create new teams. New teams cannot be created in the Microsoft Teams admin center. 
```

File: microsoft-365/enterprise/ms-cloud-germany-transition-phases.md [hunk 3]
```diff
@@ -224,16 +223,24 @@ In this phase, Skype for Business will be migrated to Microsoft Teams. Existing
...
-Connect-MicrosoftTeams -Credential $userCredential -OverridePowershellUri "https://admin4E.online.lync.com/OcsPowershellOAuth"
+Connect-MicrosoftTeams -Credential $userCredential
 ```
 
+### Known limitations until finalizing Azure AD migration
+Microsoft Teams is leveraging features of Azure AD. While the migration of Azure AD is not completed, some features of Microsoft Teams are not completely available. After phase 9, when the migration of Azure AD will be finalized, the following features become fully available:
+
+- Apps cannot be managed in the Microsoft Teams admin center.
+- New Teams can be created in the Microsoft Teams client only unless the Teams administrator constrained permissions to create new Teams for users. New Teams cannot be created in the Microsoft Teams admin center. 
+- The web version of Microsoft Teams (teams.microsoft.com) is not available.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition-phases.md]:
```suggestion
- The web version of Microsoft Teams is not available.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by denisebmsft — APPROVED** [review]
Thank you @heikobischoff @JohanFreelancer9

**Pull request merged**